styles.css

```diff
@@ -0,0 +1,108 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;500;700&display=swap");
+
+html {
+  font-family: sans-serif, 'Poppinsp';
+}
+
+body {
+  background-color: #e5e5e5;
+  height: 918px;
+  width: 375px;
+  margin: 0;
+}
+
+header {
+  background-color: white;
+}
+
+main {
+  background-image: url("images/header-shapes\ mobile.svg"), url("images/header-background-mobile\(4\).svg");
+  height: 918px;
+  width: 375px;
+  background-repeat: no-repeat;
+  padding-top: 120px;
+}
+
+.navigation {
+  text-decoration: none;
+  display: flex;
+  justify-content: space-between;
+}
+
+.logo {
+  text-decoration: none;
+  margin-top: 58px;
+  margin-left: 24px;
+  width: 76px;
+  height: 20px;
+  font-family: sans-serif, 'Poppinsp';
```

Comment:
- [OPTIONAL] I would suggest you to specify the font to your `body` selector in the styling. It is best for DRY principle and that way you don't need to specify font family for every single element. And kindly also consider to change it to `Poppins` instead of 'Poppinsp'.

Reply:
done it